[Doc update][Team clone Org-Wide support] (#21643)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/team-clone.md

Namespace: microsoft.graph

> [!NOTE]

> > Cloning isn't supported for an organization-wide team.

Reviewer: ```suggestion
> Organization-wide teams don't support cloning.
```

Create a copy of a [team](../resources/team.md). This operation also creates a copy of the corresponding [group](../resources/group.md).
You can specify which parts of the team to clone:

- **apps** - Copies Microsoft Teams apps that are installed in the team.
- **channels** – Copies the channel structure (but not the messages in the channel).
- **members** – Copies the members and owners of the group.
- **settings** – Copies all settings within the team, along with key group settings.
- **tabs** – Copies the tabs within channels.

> When tabs are cloned, they are put into an unconfigured state they are displayed on the tab bar in Microsoft Teams, and the first time you open them, you'll go through the configuration screen.

Reviewer: ```suggestion
When you clone tabs in Microsoft Teams, they initially appear in an unconfigured state on the tab bar. The first time you open these cloned tabs, you will be prompted to go through the configuration screen.
```

Namespace: microsoft.graph

> [!NOTE]

> > Cloning operation isn't supported for Organization-wide teams.

Reviewer: ```suggestion
> The cloning operation isn't supported for organization-wide teams.
```

Create a copy of a [team](../resources/team.md). This operation also creates a copy of the corresponding [group](../resources/group.md).
You can specify which parts of the team to clone:

- **apps** - Copies Microsoft Teams apps that are installed in the team.
- **channels** – Copies the channel structure (but not the messages in the channel).
- **members** – Copies the members and owners of the group.
- **settings** – Copies all settings within the team, along with key group settings.
- **tabs** – Copies the tabs within channels.

When you clone tabs in Microsoft Teams, they initially appear in an unconfigured state on the tab bar. The first time you open these cloned tabs, you will be prompted to go through the configuration screen.
(If the person opening the tab does not have permission to configure apps, they will see a message explaining that the tab hasn't been configured.)

> Cloning is a long-running operation. After the POST clone returns, you need to GET the [operation](../resources/teamsasyncoperation.md) to see if it's "running" or "succeeded" or "failed". You should continue to GET until the status is not "running". The recommended delay between GETs is 5 seconds.

Reviewer: ```suggestion
Cloning entails a time-consuming process. Once the POST clone operation concludes, retrieve the operation id from the location header in the response. Use the [Get] (../resources/teamsasyncoperation.md) method to query the status of the clone operation, which can be **running**, **succeeded**, or **failed**. Continuously perform GET requests until the status is no longer **running**. To avoid overloading the system, ensure a recommended delay of 5 seconds between successive GET requests.
```

> Cloning entails a time-consuming process. Once the POST clone operation concludes, retrieve the operation id from the location header in the response. Use the [Get] (../resources/teamsasyncoperation.md) method to query the status of the clone operation, which can be **running**, **succeeded**, or **failed**. Continuously perform GET requests until the status is no longer **running**. To avoid overloading the system, ensure a recommended delay of 5 seconds between successive GET requests.

Reviewer: ```suggestion
Cloning entails a time-consuming process. Once the POST clone operation concludes, retrieve the operation id from the location header in the response. Use the [Get](../resources/teamsasyncoperation.md) method to query the status of the clone operation, which can be **running**, **succeeded**, or **failed**. Continuously perform GET requests until the status is no longer **running**. To avoid overloading the system, ensure a recommended delay of 5 seconds between successive GET requests.
```
